composer.json

```diff
@@ -17,11 +17,11 @@
   "require": {
     "php": "^5.5.0|^7.0.0",
     "rokka/client": "^0.8.0",
-    "symfony/console" : "^3.0.0",
-    "symfony/finder" : "^3.0.0",
-    "symfony/config": "^3.0.0",
-    "symfony/dependency-injection": "^3.0.0",
-    "symfony/yaml": "^3.0.0"
+    "symfony/console" : "^2.7.0 | ^3.0.0",
```

Comment:
Please use the `||` annotation here (and below)
See: https://getcomposer.org/doc/articles/versions.md#version-range

Reply:
sorry. was not sure which one it should be and thought the wrong one would not work anymore... fixed it.